.github/workflows/test-and-publish.yaml

```diff
@@ -276,7 +275,7 @@ jobs:
           path: ./dist/
   publish:
     needs: [build-and-test, sdist]
-    runs-on: ubuntu-20.04
+    runs-on: [ubuntu-20.04,ubuntu-24.04]
```

Comment:
Why do you think you'll need to add Ubuntu 24.04 to the `publish` and `publish-nightly` jobs?

Reply:
we don't